activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+*   Deprecate `:class_name` option for the `:through` association
+
+    Through association should always relay on the :class_name of its source
```

Comment:
Spelling correction:
```suggestion
    Through association should always rely on the :class_name of its source
```

Reply:
If we can quote the options here that would be more readable I think. 

```suggestion
    The `:through` association should always rely on the `:class_name` of its source.
```